config/client.yaml

```diff
@@ -0,0 +1,56 @@
+config:
+  branches:
+    release-v1.15:
+      openShiftVersions:
+        - version: "4.15"
+          skipCron: true
```

Comment:
Is there a way to skip next OCP version. In this case 4.16, we really don't need to retest `kn` on multiple versions in the midstream.

Reply:
4.16 is only an "on demand" test and it added automatically

Reply:
Ahh ok, makes sense then.